contracts/events/TopicEvent.sol

```diff
@@ -53,8 +54,9 @@ contract TopicEvent is ITopicEvent, Ownable {
     mapping(address => bool) public didWithdraw;
 
     // Events
```

Comment:
Could you add some comments about why version number is needed in FinalResultSet()? TopicAddress is not a unique identifier of FinalResultSet (One topic address has more than one version) ?

Reply:
I can explain it here. I am trying to keep the contract as lean as possible as more KB of data (more comments), the more it costs to deploy.

On DB side, when doing a searchlog, searching for the `FinalResultSet` event originates from a `DecentralizedOracle` contract. Therefore we cannot include the `address` param in the searchlog call. So, we need the version number to know what version of TopicEvent this is.

Reply:
Discussed with Deric, version number here is to give us a filter when searchlog.